protocol/migrations/sqlite/1683725608_add_token_balances.up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS token_balances (
```

Comment:
Out of curiosity why not using appdatatabase migration?

Reply:
I think they effectively get merged in the same db, probably we should actually have everything in a single directory, since now it's not very clear (I believe they are separate only for historical reasons, since `protocol` was a separate project)
I can move them there if you prefer (I don't have a strong opinion)